ci-operator/config/openshift/release/openshift-release-master__nightly-4.7.yaml

```diff
@@ -321,6 +329,71 @@ tests:
         requests:
           cpu: 100m
     workflow: openshift-e2e-powervs-upi
+- as: e2e-cnv-deploy
```

Comment:
Usually jobs follow the pattern of `e2e-<platform>-<variation>` so I'm wondering if this shouldn't be called e2e-azure-cnv

Reply:
Jobs names were updated for all files.